### PR TITLE
Cleanup maven and remove unused stuff

### PR DESCRIPTION
**Pull request by farodin91**
* Remove bunch of maven plugins
* Remove pkg tools (Not updated and tested over multiple years) Contains stuff of Aurelius!!!

Signed-off-by: Jan Jansen <jan.jansen@gdata.de>

-----

Thank you for contributing to JanusGraph!

In order to streamline the review of the contribution we ask you
to ensure the following steps have been taken:

### For all changes:
- [ ] Is there an issue associated with this PR? Is it referenced in the commit message?
- [ ] Does your PR body contain #xyz where xyz is the issue number you are trying to resolve?
- [x] Has your PR been rebased against the latest commit within the target branch (typically `master`)?
- [x] Is your initial contribution a single, squashed commit?

### For code changes:
- [ ] Have you written and/or updated unit tests to verify your changes?
- [ ] If adding new dependencies to the code, are these dependencies licensed in a way that is compatible for inclusion under [ASF 2.0](https://www.apache.org/legal/resolved.html#category-a)?
- [ ] If applicable, have you updated the LICENSE.txt file, including the main LICENSE.txt file in the root of this repository?
- [ ] If applicable, have you updated the NOTICE.txt file, including the main NOTICE.txt file found in the root of this repository?

### For documentation related changes:
- [ ] Have you ensured that format looks appropriate for the output in which it is rendered?

### CODE REVIEW

File: janusgraph-dist/README.md [hunk 1]
```diff
@@ -31,34 +31,6 @@ The documentation output appears in:
 
 * site/
 
-## Building deb/rpm packages
```
Comment (porunov) [inline janusgraph-dist/README.md:34]: I didn't try to build dep/rpm packages but why does this functionality removed?

Comment (farodin91) [reply janusgraph-dist/README.md:34]: I went over the functionality, it is massively outdated.

If we want to have such functionality, it should be tested and update. 
(I expected we have to rebuild from stretch. => Currently, the functionality completely independent from our normal release which would require to double all scripts for both zip and system packages.)

**Comment by farodin91** [comment]
@porunov Do you like to check this PR?

**Comment by farodin91** [comment]
I will split this PR into multiple PRs.

**Pull request closed without merging**